frontend/server/app.ts

```diff
@@ -121,7 +121,10 @@ function createUIServer(options: UIConfigs) {
   registerHandler(
     app.post,
     '/apps/tensorboard',
-    getCreateTensorboardHandler(options.viewer.tensorboard.podTemplateSpec),
+    getCreateTensorboardHandler(
```

Comment:
app.test.ts has integration tests for each endpoint.
It doesn't currently cover tensorboard endpoint.

Do you want to contribute?

Reply:
Not sure I'll have time this week, but I can take a look in the next one.